### DISCUSSION TITLE
Ошибка при установки VKUI.

### DISCUSSION THREAD

**Discussion by Mityacatmc** [VKCOM/VKUI#5494] [Q&A] [answered]
### Описание

Пытаюсь установить VKUI. А выходит данная ошибка (на скрине) : 
![IMG_20230717_230513_269](https://github.com/VKCOM/VKUI/assets/124724954/222eae8e-e532-4f6c-9d41-da1d57821e90)

### Версия

NODEJS VERSION 13

### Шаги воспроизведения

Cd (путь)
npm i @vkontakte/vkui @vkontakte/icons @vkontakte/bridge

**Comment by SevereCloud** [comment]
Привет, как написано в логах попробуй удалить лок файл или запусти npm с флагом `--force`

**Comment by Mityacatmc** [comment]
Сейчас попробую.

**Comment by Mityacatmc** [comment]
Assesrtion failed: new_time >= loop->time, file c:\ws\deps\uv\src\win\core.c, line 309

**Comment by SevereCloud** [comment]
Попробуй также обновить версию nodejs и npm

**Comment by Mityacatmc** [comment]
Хорошо

**Comment by Mityacatmc** [comment]
Ничего не изменилось

**Reply by SevereCloud** [reply]
Что показывают следующие команды?

```cmd
node -v
npm -v
```

**Reply by Mityacatmc** [reply]
Npm version: 6.14.4
NODEJS VERSION : 13.14.0.
У меня Windows 7.

**Reply by SevereCloud** [reply]
А что насчет обновления nodejs и npm?

**Reply by Mityacatmc** [reply]
На Windows 7 смог установить nodejs 13.14.0.
14.x.x не установить.
Npm вроде обновил

**Reply by Mityacatmc** [reply]
Жду

**Reply by SevereCloud** [reply, marked as answer]
К сожалению тут уже ничем более помочь не могу. Эти версии nodejs и Npm устарели

**Comment by Mityacatmc** [comment]
Установил, спасибо за поддержку.
